scalableminds/webknossos
# Fix editable text style

This PR mainly fixes some small line breaks in the some places in the UI that I found annoying.

Changes:
- Annotation Info Tab
  - Ensure annotation name and pencil icon are on the same line
  - Ensure annotation description and pencil icon are on the same line
- Segments Tab
  - Ensure each list entry is on one line
- Refactored some typescript typing and removed suppressions/increased coverage

Before: 
<img width="367" alt="image" src="https://user-images.githubusercontent.com/1105056/216055207-205d527b-1b99-4fca-9a56-6c6b397168b6.png">

After:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/1105056/216055108-c42c7eea-3c62-46d8-8802-f063042894f3.png">

Before:
<img width="304" alt="image" src="https://user-images.githubusercontent.com/1105056/216056593-809ea050-cfa6-49b4-9bd3-d933c008d520.png">

After:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/1105056/216055024-c4661643-0c8c-41f4-b43c-0170fb6b966d.png">


Layout/CSS tested in Safari 16.03, Chrome 109.0.5414.119, Firefox 109.0.1 on MacOS.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Check that the line breaks work in your browser
- Double-check where else the changed components might be use, .e.g. dataset list on dashboard

### Issues:
- None

------
(Please delete unneeded items, merge only when none are left open)
